unity-ggjj/Assets/Scripts/NarrativeScript/NarrativeScriptPlayer/NarrativeScriptPlayer.cs

```diff
@@ -48,11 +45,11 @@ public bool CanPressWitness
                 return _subNarrativeScript.CanPressWitness;
             }
 
-            return IsAtChoice && GameMode == GameMode.CrossExamination && !_appearingDialogueController.IsPrintingText;
+            return IsAtChoice && GameMode == GameMode.CrossExamination && !_narrativeGameState.AppearingDialogueController.IsPrintingText && !Waiting;;
```

Comment:
Nit: Double semicolon.
```suggestion
            return IsAtChoice && GameMode == GameMode.CrossExamination && !_narrativeGameState.AppearingDialogueController.IsPrintingText && !Waiting;
```